Packages/com.unity.ide.rider/Rider/Editor/ProjectGeneration/ProjectGeneration.cs

```diff
@@ -895,7 +895,7 @@ private static string GenerateAnalyserRuleSet(string[] paths)
         if (!paths.Any())
             return string.Empty;
       
-        return string.Join("\r\n", paths.Select(a => $"\r\n  <CodeAnalysisRuleSet>{a}</CodeAnalysisRuleSet>"));
+        return $"{Environment.NewLine}{string.Join(Environment.NewLine, paths.Select(a => $"  <CodeAnalysisRuleSet>{a}</CodeAnalysisRuleSet>"))}";
```

Comment:
I have asked around and it seems those "\r\n" instead of Environment.NewLine are there for a reason.
Could you please revert them back.

Reply:
Will do